components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/internal/OrganizationManagementDataHolder.java

```diff
@@ -77,4 +87,19 @@ public void setOrganizationManagerListener(OrganizationManagerListener organizat
 
         this.organizationManagerListener = organizationManagerListener;
     }
+
+    public DataSource getDataSource() {
+
+        return dataSource;
+    }
+
+    public void initDataSource() {
+
+        try {
+            this.dataSource = DatabaseUtil.getRealmDataSource(CarbonContext.getThreadLocalCarbonContext().
+                    getUserRealm().getRealmConfiguration());
+        } catch (UserStoreException e) {
+            LOG.error("Error while retrieving user management data source", e);
```

Comment:
-> "Error while retrieving user management data source." Add a full-stop at the end

Reply:
Fixed by - https://github.com/wso2-extensions/identity-organization-management-core/pull/36